rtflib/tests/helloworld-multi.rtf

```diff
@@ -0,0 +1,17 @@
+{\rtf1\ansi\deff0
+{
+\colortbl;
+\red255\green0\blue0;
+\red0\green255\blue0;
+}
+hello world. A Line of text.\
+\ 
```

Comment:
not sure if there's a better way to format these so output has less line breaks...

Reply:
You can open an issue documenting this if you're not sure how to fix it but think it might be worth addressing later (or worth someone else addressing)

Reply:
will do